components/event-sources/test/fixtures/ksvc.json

```diff
@@ -30,6 +30,9 @@
     "spec": {
         "template": {
             "metadata": {
+                "labels": {
+                    "kyma-project.io/event-mesh": "true"
```

Comment:
Same comment about the empty string instead of that misleading "true" value (we only care that the label _exists_).